kano_i18n/init.py

```diff
@@ -48,6 +54,9 @@ def register_domain(domain, locale_dir=None):
         DOMAINS_TO_REGISTER.append(
             (domain, locale_dir)
         )
+        # and register stubs for _ and N_
+        __builtin__.__dict__['_'] = lambda msg: msg
+        __builtin__.__dict__['N_'] = lambda msg: msg
```

Comment:
Not sure what the best value for the `errors` param should be:

```
In [14]: u'¡Hola!'.encode('ascii', 'ignore')
Out[14]: 'Hola!'

In [15]: u'¡Hola!'.encode('ascii', 'replace')
Out[15]: '?Hola!'

In [16]: u'¡Hola!'.encode('ascii', 'xmlcharrefreplace')
Out[16]: '&#161;Hola!'
```

@Ealdwulf @keberlin what do you think?